src/less/mainNav.less

```diff
@@ -172,16 +169,13 @@ main-nav {
       }
     }
 
-
-
     &.selected {
       .btn {
         background-color: @main-nav-active-bg;
-        box-shadow: @main-nav-selected-highlight;
-        border: 1px solid @main-nav-selected-highlight;
```

Comment:
Seems like adding the border: 0; outside of the .selected scope on .main-nav-menu-items .btn worked, so going with that solution